skops/cli/_update.py

```diff
@@ -0,0 +1,89 @@
+from __future__ import annotations
+
+import argparse
+import json
+import logging
+import zipfile
+from pathlib import Path
+from typing import Optional, Union
+
+from skops.cli._utils import get_log_level
+from skops.io import dump, load
+from skops.io._protocol import PROTOCOL
+
+
+def _update_file(
+    input_file: Union[str, Path],
+    output_file: Union[str, Path],
+    logger: logging.Logger = logging.getLogger(),
+) -> None:
+    """Function that is called by ``skops update`` entrypoint.
+
+    Loads a skops model from the input path, converts to the current skops format, and
+    saves to output file.
+
+    Parameters
+    ----------
+    input_file : Union[str, Path]
+        Path of input skops model to load.
+
+    output_file : Union[str, Path]
+        Path to save the updated skops model to.
+
+    """
+    input_model = load(input_file, trusted=True)
+    with zipfile.ZipFile(input_file, "r") as zip_file:
+        input_file_schema = json.loads(zip_file.read("schema.json"))
+
+    if input_file_schema["protocol"] != PROTOCOL:
+        dump(input_model, output_file)
+        logger.info(f"Updated skops file written to {output_file}")
+
+    logger.info(f"Input file is already up to date to the current protocol: {PROTOCOL}")
```

Comment:
```suggestion
    logger.info(f"Input file is already up to date to the current protocol: {PROTOCOL}")
```

Should this not be in the `else` clause? Also, I prefer a wording that says what was actually done, something like "File was not updated because ...".